package.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Is this file added on purpose?

Reply:
File is needed for https://github.com/ScottBrenner/generate-changelog-action